### ISSUE TITLE
Renaming a new note that was made after search

### ISSUE THREAD

**Issue by chanclr** [standardnotes/forum#907]
If I make a search, then create a new note, then the note title is the search term. I recognize that this is [a new feature](https://github.com/standardnotes/web/pull/367#issue-374958339)
But if I go to change the title, then the note disappears from view, and sometimes I would have to scroll up my list of notes to get to the note again.

**Comment by JaspalSuri** [comment]
I can't seem to reproduce the part regarding the note disappearing. It does disappear from the search results, depending on how different the text contained in the title compares to your search query. 

Here's what I did:

1. Searched for "cool dogs"
2. Clicked on the plus button, which resulted in a new note with the title "cool dogs"
3. Changed the title to "cool alpacas" 
4. The note disappears from the list of results. To find it again, I would have to exit out of search.

Depending on the way you've sorted your notes, you might not see it at the top. E.g. if you're not viewing all of your notes (tagged and untagged) and if you've enabled `reverse sort` and have it appear at the bottom.

Let me know if that helps!
